tox.ini

```diff
@@ -12,25 +12,26 @@ envdir =
     pypy3: {toxworkdir}/pypy3
     py: {toxworkdir}/py
 commands =
-    python housekeep.py setup
-    nocov: python -m nose2 -v {posargs}
-    cov: python -m coverage run -m nose2
-    cov: python -m coverage combine
-    cov: python -m coverage html --directory=htmlcov/{envname}
-    cov: python -m coverage report -m --fail-under=100
-    cov: python -m coverage xml
-    diffcov: diff-cover coverage.xml --html-report diffcov.html
-    diffcov: diff-cover coverage.xml --fail-under=100
-    profile: python -m nose2 --profile {posargs}
+    python housekeep.py prep
+    nocov: pytest --verbose -p no:cov {posargs}
+    cov: pytest --cov --cov-report=xml --cov-report=html --cov-report=term {posargs}
+    diffcov: diff-cover coverage-{env:INTERP}.xml --html-report diffcov-{env:INTERP}.html
+    diffcov: diff-cover coverage-{env:INTERP}.xml --fail-under=100
+    profile: pytest --profile {posargs}
     python housekeep.py gather
 #sitepackages = True
 usedevelop = True
 deps =
-    nose2
-    coverage
+    packaging
+    pytest
+    pytest-cov
+    pytest-mock
+    pytest-profiling
+    pytest-sugar
```

Comment:
Though `packaging` and `pytest-mock` are not (yet) used in the test cases, they _are_ used in PR #202 

We add them here to iron out possible glitches before the huge #202 gets merged.